### DISCUSSION TITLE
Column values

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6406] [Help]
Helloe everyone.
How to get the list of all values ​​in a column

**Comment by melohagan** [comment]
This https://docs.budibase.com/docs/formula#tutorial-test-scores-using-sum 
without the sum

E.g. `{{ pluck TableName 'ColumnName' }}` will list all ColumnName values comma separated. 

There's a JavaScript example in there too.

**Comment by rafikApp** [comment]
I want to compare the value entered in the form with each value in the model column list
but it didn't work 

![Capture](https://user-images.githubusercontent.com/106697208/174787067-f0847f0e-fcc3-43cb-8335-24723df4bfa8.PNG)
